### PR TITLE
Improve Apply Patch page

### PR DESCRIPTION
**Pull request by thiaramus**
## Purpose of this pull request

This pull request (PR) is to separate patch application methods from one single page to individual pages.
MQP section is rather large and it would be nice to have it on a separate page, alongside the composer and command line patch application methods.

The work was done in the scope of the JIRA story: https://jira.corp.magento.com/browse/MC-37903

Certain Help Center pages are currently pointing to the https://devdocs.magento.com/guides/v2.4/comp-mgr/patching.html and https://devdocs.magento.com/guides/v2.4/comp-mgr/patching.html#mpq, the change should not break the links, but I will work on updating the related articles to point directly to the new MQP page after the PR is merged.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/comp-mgr/patching.html
- https://devdocs.magento.com/quality-patches/release-notes.html
- https://devdocs.magento.com/cloud/project/project-patch.html

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@thiaramus thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: src/cloud/project/project-patch.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ functional_areas:
 Both packages improve the integration of all {{site.data.var.ee}} versions with Cloud environments and support quick delivery of critical, optional, and custom fixes. You can use these packages to apply, revert, and view general information about all individual patches that are available for Magento.
 
 {:.bs-callout-tip}
-You can use the [Magento Quality Patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching.html#mqp) and [{{ site.data.var.mcp-prod }}](#standalone) packages as stand-alone packages for {{ site.data.var.ce }} and {{ site.data.var.ee }} projects. We recommend using the Magento Quality Patches package for non-Cloud projects.
+You can use the [Magento Quality Patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching/mqp.html#mqp) and [{{ site.data.var.mcp-prod }}](#standalone) packages as stand-alone packages for {{ site.data.var.ce }} and {{ site.data.var.ee }} projects. We recommend using the Magento Quality Patches package for non-Cloud projects.
```
Comment (viktym) [inline src/cloud/project/project-patch.md:17]:
```suggestion
You can use the [Magento Quality Patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching/mqp.html) and [{{ site.data.var.mcp-prod }}](#standalone) packages as stand-alone packages for {{ site.data.var.ce }} and {{ site.data.var.ee }} projects. We recommend using the Magento Quality Patches package for non-Cloud projects.
```

File: src/cloud/project/project-patch.md [hunk 2]
```diff
@@ -205,7 +201,7 @@ To apply and test a custom patch on a Cloud environment:
 
 ## Apply patches to a non-Cloud project {#standalone}
 
-Use the [Magento Quality Patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching.html#mqp) package for {{ site.data.var.ce }} and {{ site.data.var.ee }} projects.
+Use the [Magento Quality Patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching/mqp.html#mqp) package for {{ site.data.var.ce }} and {{ site.data.var.ee }} projects.
```
Comment (viktym) [inline src/cloud/project/project-patch.md:204]:
```suggestion
Use the [Magento Quality Patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching/mqp.html) package for {{ site.data.var.ce }} and {{ site.data.var.ee }} projects.
```

File: src/guides/v2.3/comp-mgr/patching/mqp.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+The [MQP package][] delivers individual patches developed by Magento and allows you to apply, revert, and view general information about all individual patches that are available for the installed version of {{ site.data.var.ee }} or {{ site.data.var.ce }}.
+
+{:.bs-callout-warning}
+We do not recommend using the MQP package to apply large numbers of patches because it increases the complexity of your code, which makes upgrading to a new version of Magento more difficult.
+
+#### Install the MQP package
+
+{:.bs-callout-info}
+If it is not already installed, you must install [Git](https://github.com/git-guides/install-git) before installing the MQP package.
```
Comment (viktym) [inline src/guides/v2.3/comp-mgr/patching/mqp.md:18]:
```suggestion
If it is not already installed, you must install [Git](https://github.com/git-guides/install-git) or [Patch](https://man7.org/linux/man-pages/man1/patch.1.html) before installing the MQP package.
```

File: src/quality-patches/release-notes.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ The [Magento Quality Patches](https://github.com/magento/quality-patches) packag
 -  {:.bug}Known issues -->
 
 {:.bs-callout-info}
-See [Apply patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching.html#mqp) for instructions on applying patches to your Magento projects.
+See [Apply patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching/mqp.html#mqp) for instructions on applying patches to your Magento projects.
```
Comment (viktym) [inline src/quality-patches/release-notes.md:19]:
```suggestion
See [Apply patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching/mqp.html) for instructions on applying patches to your Magento projects.
```

File: src/guides/v2.3/comp-mgr/patching/command-line.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
+group: software-update-guide
+title: Apply patches using Command Line tools
+functional_areas:
+  - Upgrade
+---
+
+### Command line
+
+1. Upload the local file into the `<Magento_root>` on the server using FTP, SFTP, SSH or your normal transport method.
```
Comment (meker12) [inline src/guides/v2.3/comp-mgr/patching/command-line.md:10]:
```suggestion
1. Upload the local file into the `<Magento_root>` directory on the server using FTP, SFTP, SSH or your normal transport method.
```

File: src/guides/v2.3/comp-mgr/patching/mqp.md [hunk 6]
```diff
@@ -0,0 +1,201 @@
+---
+group: software-update-guide
+title: Apply patches using Magento Quality Patches Tool (MQP)
+functional_areas:
+  - Upgrade
+---
+
+### Magento Quality Patch (MQP) package {#mqp}
+
+The [MQP package][] delivers individual patches developed by Magento and allows you to apply, revert, and view general information about all individual patches that are available for the installed version of {{ site.data.var.ee }} or {{ site.data.var.ce }}.
```
Comment (meker12) [inline src/guides/v2.3/comp-mgr/patching/mqp.md:10]:
```suggestion
The [Magento Quality Package (MQP) package][] delivers individual patches developed by Magento and allows you to apply, revert, and view general information about all individual patches that are available for the installed version of {{ site.data.var.ee }} or {{ site.data.var.ce }}.
```

File: src/guides/v2.3/comp-mgr/patching/mqp.md [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+   {:.bs-callout-info}
+   When you run the `status` command, the patches that where included in the new version are no longer displayed in the table of available patches.
+
+#### Logging
+
+The MQP package logs all operations in the `<Magento_root>/var/log/patch.log` file.
+
+<!-- Link Definitions -->
+
+[MQP package]: https://github.com/magento/quality-patches
```
Comment (meker12) [inline src/guides/v2.3/comp-mgr/patching/mqp.md:201]:
```suggestion
[Magento Quality Package (MQP) package]: https://github.com/magento/quality-patches
```

File: src/guides/v2.3/comp-mgr/patching/composer.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+functional_areas:
+  - Upgrade
+---
+
+### Composer
+
+{:.bs-callout-warning}
+Always perform comprehensive testing before deploying any custom patch.
+
+To apply a custom patch using Composer:
```
Comment (meker12) [inline src/guides/v2.3/comp-mgr/patching/composer.md:13]:
```suggestion
{.:procedure}
To apply a custom patch using Composer:
```

**Review by meker12 — CHANGES_REQUESTED** [review]

**Review by meker12 — APPROVED** [review]

**Pull request merged**